src/applications/disability-benefits/all-claims/pages/militaryHistory.js

```diff
@@ -54,14 +70,44 @@ export const uiSchema = {
         },
       },
     },
+    'view:separationLocation': {
+      'ui:title': SeparationLocationTitle,
+      'ui:description': SeparationLocationDescription,
+      'ui:options': {
+        hideIf: formData =>
+          environment.isProduction() ||
+          !isBDD(formData.serviceInformation.servicePeriods),
```

Comment:
This will always show up in production? Should this be an `&&` instead of an `||`?

Reply:
It's hideIf, so it will hide the field if in production, so I think I have my logic right here.

Reply:
derp, you're right... `hideIf` 😛